models/User.js

```diff
@@ -53,7 +53,7 @@ UserSchema.methods.toProfileJSONFor = function(user){
   return {
     username: this.username,
     bio: this.bio,
-    image: this.image || 'https://static.productionready.io/images/smiley-cyrus.jpg',
+    image: this.image || '/public/alien_avatar.png',
```

Comment:
I'm not sure if it's correct to use a string here, maybe it should be imported as a path to this file?